lib/purchases_flutter.dart

```diff
@@ -159,8 +163,15 @@ class Purchases {
   /// [customerInfoUpdateListener] CustomerInfo update listener.
   static void addCustomerInfoUpdateListener(
     CustomerInfoUpdateListener customerInfoUpdateListener,
-  ) =>
-      _customerInfoUpdateListeners.add(customerInfoUpdateListener);
+  ) {
+    _customerInfoUpdateListeners.add(customerInfoUpdateListener);
+    if (_customerInfoUpdateListeners.length == 1) {
+      final lastReceivedCustomerInfo = _lastReceivedCustomerInfo;
+      if (lastReceivedCustomerInfo != null) {
+        customerInfoUpdateListener(lastReceivedCustomerInfo);
+      }
+    }
+  }
```

Comment:
I haven't added tests yet. I wanted to validate if we're ok with this approach first.

Reply:
Yup this makes sense 👍🏻